### DISCUSSION TITLE
Login page loop.

### DISCUSSION THREAD

**Discussion by berma83** [Budibase/budibase#2375] [Help] [answered]
Hi, 

After a docker service restart, I can't login anymore, bb always redirects me to the login page.

version: 0.9.105
[log.txt](https://github.com/Budibase/budibase/files/6997470/log.txt)

https://user-images.githubusercontent.com/53205779/129664383-72376e55-a0ac-405b-9339-7e6244e624eb.mp4

**Comment by shogunpurple** [comment, marked as answer]
Hi @berma83 - this was the result of a merge that went in yesterday and has since been reverted. Please update your budibase and you should be up and running again

**Reply by KernOil** [reply]
@shogunpurple I am having the same problem. how do I update my docker container without losing any data?

**Reply by berma83** [reply]
Hi @shogunpurple, use these commands to update:

`docker pull budibase/worker`
`docker pull budibase/apps`

**Reply by KernOil** [reply]
@berma83 Thank you sir!
